datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -671,34 +671,6 @@ def submit_histogram_bucket(
             flush_first_value,
         )
 
-    def database_monitoring_query_sample(self, raw_event):
```

Comment:
Just a small thing, in order to be able to merge this and not break, we need to keep these methods for backward compatibility with the existing checks. When we migrate the checks then we can clean these up.